.github/workflows/tests.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+      env: EXTRA_PACKAGES=libpcap-dev
```

Comment:
Please add that directly to tests/run-alpine instead.

Reply:
Yeah, I realised that now, trying to fix that up currently :-)